devbox.json

```diff
@@ -1,6 +1,6 @@
 {
   "packages": [
-    "go_1_22@latest",
```

Comment:
I think we should keep this pinned to 1.22 if we can as upgrades past 1.22 will likely need some more intentional work to do